Makefile

```diff
@@ -19,18 +20,29 @@ JS_TESTING_FILES := $(wildcard test/test*.js test-whitelist/test*.js)
 NODE ?= node
 NPM ?= $(NODE) $(shell which npm)
 MOCHA ?= $(NODE) $(BIN)/mocha
+BABEL ?= $(NODE) $(BIN)/babel
 WEBPACK ?= $(NODE) $(BIN)/webpack
 
-standalone: build/wpcom.js
+build: mkbuilddir
+	$(BABEL) index.js --optional runtime --out-dir $(BUILDDIR)
+	$(BABEL) lib --optional runtime --out-dir $(BUILDDIR)/lib
 
-build/wpcom.js: $(COMPILED_FILES)
+mkbuilddir:
```

Comment:
Similarly, `make mkbuilddir` will _always_ run the `mkdir -p` command, even when it already exists, since it doesn't depend on `$(BUILDDIR)` on the right-hand side.